Progress/Schedule.md

```diff
@@ -0,0 +1,114 @@
+# Camp Day 1
+### 12:15pm - 12:30pm
+Drop off
+
+### 12:30pm - 1:00pm
+Introductions, getting settled and talking about the activites for today
+
+### 1:00pm - 2:00pm
+Making ALL necessary textures for today (Ingot, tool, block, and armor if they finished early)\
+
+### 2:00pm - 2:30pm
+Creating GUI and the Block
+
+### 2:30pm - 3:00pm
+(stem activity) Activity one, minecraft bingo! ( snack might also go here)
+ 
+### 3:00pm - 4:00pm
+Creating ingot, ingot recipe and the Multitool (possibly tool recipe)
+
+### 4:00pm - 4:30pm
+Activity (Minecreaft pictionary/ pick up)
```

Comment:
call this 'overflow time' - time for the kids to do something while waiting to be picked up